plugins/catalog-backend/src/modules/core/UrlReaderProcessor.ts

```diff
@@ -124,6 +124,23 @@ export class UrlReaderProcessor implements CatalogProcessor {
   ): Promise<{ response: { data: Buffer; url: string }[]; etag?: string }> {
     // Does it contain globs? I.e. does it contain asterisks or question marks
     // (no curly braces for now)
+    const getUrlReaderGCs = location.startsWith(
```

Comment:
I don't think that this logic should live here.

I think that the cause of this issue is actually something to do with the below line using 

`const { filepath }  = parseGitUrl(location)` and the location not being an actual Git URL, it's a google domain instead.

@Rugvip @freben I think we had some thoughts about this, could maybe suggest a better way to do this?

/cc @backstage/catalog-maintainers

Reply:
I think that this comment is the fix that we actually want @armandocomellas1:

https://github.com/backstage/backstage/issues/21955#issuecomment-1874175627

Reply:
> I think that this comment is the fix that we actually want @armandocomellas1:
> 
> [#21955 (comment)](https://github.com/backstage/backstage/issues/21955#issuecomment-1874175627)

@benjdlambert Actually if you console.log(filepath) it bring just the links that are inside a local backstage templeate, so to be honest l think this solution is not the fix that we want  if (filepath?.match(/[*?]/)) { } because the condition never mets. because the variable filepath does not bring the Google Domain URL paths.

Reply:
@benjdlambert Thank you so much for your quick response and YES! it actually works with that line of code. so let me change it and update this PR.

I really appreciate your help @benjdlambert @viglesiasce